content/SCALE/SCALETutorials/Network/SettingUpIPMI.md

```diff
@@ -7,32 +7,53 @@ tags:
 - ipmi
 ---
 
-{{< hint type=note >}}
-IPMI requires compatible hardware! Refer to your hardware documentation to determine if the TrueNAS web interface has IPMI options.
-{{< /hint >}}
+{{< include file="/static/includes/IPMICompatibility.md" >}}
 
-Many [TrueNAS Storage Arrays](https://www.truenas.com/docs/hardware/) have a built-in out-of-band management port that provides side-band management should the system become unavailable through the web interface.
+Many [TrueNAS systems](https://www.truenas.com/docs/hardware/) have a built-in out-of-band management port that provides side-band management should the system become unavailable through the web interface.
```

Comment:
I think this sentence, though well written, might be hard to parse or translate for ESL readers. That said, I don't know how to rephrase this to make it easier. Thoughts?

Reply:
How about 
```suggestion
Many [TrueNAS systems](https://www.truenas.com/docs/hardware/) include a built-in out-of-band management port, enabling system access even when the web interface is unavailable.
```